Content.Client/HealthAnalyzer/UI/HealthAnalyzerWindow.xaml.cs

```diff
@@ -109,17 +112,20 @@ public void Populate(HealthAnalyzerScannedUserMessage msg)
 
             DrawDiagnosticGroups(damageSortedGroups, damagePerType);
 
-            if (_entityManager.TryGetComponent(target, out HungerComponent? hunger)
-                && hunger.StarvationDamage != null
-                && hunger.CurrentThreshold <= HungerThreshold.Starving)
+            if (_entityManager.TryGetComponent(target, out SatiationComponent? satiationComponent))
             {
-                var box = new Control { Margin = new Thickness(0, 0, 0, 15) };
+                foreach (var (id, satiation) in satiationComponent.Satiations)
+                {
+                    if (satiation.CurrentThresholdDamage == null) continue;
 
-                box.AddChild(CreateDiagnosticGroupTitle(
-                    Loc.GetString("health-analyzer-window-malnutrition"),
-                    "malnutrition"));
+                    var box = new Control { Margin = new Thickness(0, 0, 0, 15) };
```

Comment:
maybe you want to use a V/HSeparator?

Reply:
Woops, now box creation is outside of loop, and satiation levels with damage warrnings are not diagnostic group titles but diagnostic item labels.